src/components/InfoPopup/InfoPopup.js

```diff
@@ -22,12 +22,12 @@ function InfoPopup({ name, isOpen, onClose, errStatus }) {
     }, [onClose])
```

Comment:
`   }, [onClose])`

you need to watch `isOpen` as well here

```js

 }, [onClose, isOpen])

```

and add listeners only when the popup is opened


```js

    useEffect(() => {
        if (!isOpen) return; <== here you prevent useEffect from triggering if not open
        const close = (e) => {

```

the same thing in `PopupWithForm`